arxiv/metadata/metacheck.py

```diff
@@ -103,15 +147,13 @@ def long_word_caps(s: str) -> bool:
     Returns true (was: a list of long words) if s appears to have
     two or more words which are
     * at least 6 characters long,
-    * in all caps,
-    * contain at least one capitalized letter (not digits or punctuation!), 
+    * is in all caps (TODO: strip trailing comma ","?)
     * and which are not in our list of KNOWN_WORDS_IN_ALL_CAPS.
     """
     num_matches = 0
     for word in s.split():
         if len(word) >= 6 and \
-           word.upper() == word and \
-           word.lower() != word and \
+           re.match( "^[A-Z]*$", word ) and \

```

Comment:
What are cases where `word.upper() == word` is not enough for this test?

Reply:
if word = "123.456", word.upper() == word , but that's not something this code wants to complain about.